multires_image/package.xml

```diff
@@ -13,11 +13,10 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <depend>libopencv-dev</depend>
+  <build_depend>libqt5-opengl-dev</build_depend>
```

Comment:
I find the package dependencies easier to parse when sorted by package, rather than dependency type.